scripts/badge/LowLevelWriter.mjs

```diff
@@ -0,0 +1,89 @@
+"use strict";
+
+import fs from 'fs';
+import { EventEmitter, once } from 'events';
+
+/**
+ * File writer class that ensures that no data will be lost.
+ * Ref https://stackabuse.com/writing-to-files-in-node-js/
+ * Imported from our personal massCode snippets db, and written
+ * to fix the most frustrating bug in my PhD data processing code....
+ * @extends EventEmitter
+ */
+class LowLevelWriter extends EventEmitter {
+	constructor() {
+		super();
+		
+		this.fd = null;
+		this.is_writing = false;
```

Comment:
As a JS dev :disappointed: the standard in JS is `camelCase`. Why you choose snake_case?
```suggestion
		this.isWriting = false;
```

Reply:
I was not aware of this. I've always used snake_case? is this a particular issue here?

Reply:
No, just the convention.